sim/src/main/cc/firesim/firesim_top.cc

```diff
@@ -331,6 +332,49 @@ uint64_t host_mem_offset = -0x80000000LL;
     #endif
 #endif
 
+#ifdef GROUNDTESTBRIDGEMODULE_struct_guard
```

Comment:
Technically you only need this you're going to super-node GroundTest but this is safe.